be/src/olap/delta_writer.h

```diff
@@ -120,6 +120,8 @@ class DeltaWriter {
 
     int64_t tablet_id() { return _tablet->tablet_id(); }
 
+    int64_t txn_id() { return _req.txn_id; }
```

Comment:
Maybe we should not add this interface just for printing logs

Reply:
I think it is necessary, so do int64_t tablet_id() { return _tablet->tablet_id(); } above.